packages/fuels/src/setup-launch-node-server.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Provider } from '@fuel-ts/account';
+import { waitUntilUnreachable } from '@fuel-ts/utils/test-utils';
+import { spawn } from 'node:child_process';
+
+function startServer(
+  port: number = 0
+): Promise<{ serverUrl: string; killServer: () => void } & Disposable> {
```

Comment:
We should probably define the return type as an interface 

```ts
interface ServerInfo extends Disposable {
  serverUrl: string;
  killServer: () => void;
}
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2378/commits/7a1ebbfe1a2944f5aebf3de9e75bd6003ba8b497